lldp_util.c

```diff
@@ -1055,9 +1055,11 @@ int get_saddr6(const char *ifname, struct sockaddr_in6 *saddr)
 
 	rc = getifaddrs(&ifaddr);
 	if (rc == 0) {
+		rc = -1;
 		for (ifa = ifaddr; ifa != NULL; ifa = ifa->ifa_next) {
-			if ((ifa->ifa_addr->sa_family == AF_INET6) &&
-			    (strncmp(ifa->ifa_name, ifname, IFNAMSIZ) == 0)) {
```

Comment:
The open bracket is deleted but the corresponding closing bracket isn't touched. Should the open bracket be added to one of the other lines?

Reply:
Yes.  I'll fix it as I address the other comment.